resources/assets/tconstruct/lang/de_de.lang

```diff
@@ -826,6 +856,15 @@ ui.mininglevel.cobalt=Cobalt
 # Death messages
 death.attack.smeltery=%s hat versucht, einen geschmolzenen Spieler zu erschaffen
 death.attack.splinter=%s hat sich einen Splitter eingefangen. War anscheinend ein großer
+death.attack.bleed=%1$s ist durch %2$s Angriff verblutet
+death.attack.bleed.item=%1$s verblutete durch %2$s Angriff mit %3$s
+
+message.mending_moss.not_enough_levels=Reparierendes Moos benötigt mindestens %s Stufen
+tconstruct.channel.connected.in=Setze Seite um nach innen zu fließen
```

Comment:
Still not really happy with those.. but I'll see if I can find something. Otherwise it's fine.